### ISSUE TITLE
What are the available options for localisation?

### ISSUE THREAD

**Issue by mohamedluay** [yelog/hexo-theme-3-hexo#72]
I really liked the theme and want to use it for my blog which will be in english. Have any one managed to so from your side? or do you have any localisation mechanism that I can use to provide an english version from this and make a pr for it?

**Comment by yelog** [comment]
Glad you like this theme, I will provide English version in a week.

**Comment by mohamedluay** [comment]
I am interested in contributing to this localisation. I have already forked the repo and started adding the language folder with both en.yml and zh-CN.yml files. I can submit a pr for the localisation if that would help. However I have a question to you, after applying the translation to your keyword, I found that the english keywords are a little bit larger in size and hence messing up the styling (Like the image below 👇), what is the approach that you do recommend to handle that?
![Screen Shot 2020-02-05 at 9 21 51 PM](https://user-images.githubusercontent.com/1096736/73875603-2f8b3780-485e-11ea-8a73-bc1b454c0898.png)
